src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -2951,6 +2950,13 @@ private void OnProjectStarted(object sender, ProjectStartedEventArgs e)
             });
         }
 
+        internal void EnableBuildCheck()
```

Comment:
Doc comment about why/when this should be used please (in general you'd want to enable buildchecks through the `BuildParameters` but this is for replay)

Reply:
done